### ISSUE TITLE
Training Loss problem 

### ISSUE THREAD

**Issue by MillenniumFalcon1097** [BinWang-shu/pytorch_hed#7]
Hi, I'm training the model on BSDS500 dataset using the pretrained VGG16 state_dict, however I find the loss printed is all 0.000. What's the problem with it? Looking forward to your reply!
